lib/razorpay/payment.rb

```diff
@@ -10,6 +10,46 @@ class Payment < Entity
     def self.request
       Razorpay::Request.new('payments')
     end
+    
+    def self.createRecurringPayment(data={})
+      request.post "create/recurring" , data
+    end
+
+    def self.createJsonPayment(data={})
+      request.post "create/json" , data
+    end
+
+    def self.fetchPaymentDowntime()
+      request.get "downtimes"
+    end
+
+    def self.fetchPaymentDowntimeById(id)
+      request.get "downtimes/#{id}"
+    end
+
+    def self.fetchCardDetails(id)
+      request.get "#{id}/card"  
+    end
+    
+    def fetchTransfer
+      self.class.request.get "#{id}/transfers"  
+    end
+
+    def self.fetchRefund(id, refundId)
```

Comment:
shouldnt params be same as https://github.com/razorpay/razorpay-php/blob/master/documents/refund.md#fetch-a-specific-refund-for-a-payment

Reply:
Done